### PR TITLE
DA-563 fixing fake participant importer

### PR DESCRIPTION
**Pull request by Seeker1911**

### CODE REVIEW

File: rest-api/tools/auth_setup.sh [hunk 1]
```diff
@@ -61,7 +61,7 @@ source ${REPO_ROOT_DIR}/tools/setup_vars.sh
 TMP_DB_INFO_FILE=${TMP_DIR}/db_info.json
 PORT=3308
 CLOUD_PROXY_PID=
-
+export "GOOGLE_APPLICATION_CREDENTIALS"=$CREDS_FILE
```
Comment (danqrodney) [inline rest-api/tools/auth_setup.sh]: Nit: I don't think you need the quotes here.

File: rest-api/tools/import_organizations.py [hunk 2]
```diff
@@ -584,7 +584,7 @@ def main(args):
   HPODao()._invalidate_cache()
   OrganizationImporter().run(args.organization_file, args.dry_run)
   OrganizationDao()._invalidate_cache()
-  SiteImporter().run(args.site_file, args.dry_run)
+  SiteImporter().run(args.site_file, args.dry_run, args.creds_file)
```
Comment (danqrodney) [inline rest-api/tools/import_organizations.py]: I don't think you should pass in creds_file anymore? Does this actually work?

File: rest-api/tools/auth_setup.sh (hunk 1 above)
Comment (Seeker1911) [reply rest-api/tools/auth_setup.sh]: You're right

File: rest-api/tools/import_organizations.py (hunk 2 above)
Comment (Seeker1911) [reply rest-api/tools/import_organizations.py]: Nope, fixed.

**Pull request merged**